core_api/src/lib.rs

```diff
@@ -49,7 +49,7 @@
 //! hc.stop().expect("couldn't stop the app");
 //!
 //!```
-
+#![feature(try_from)]
```

Comment:
I feel like I shouldn't have to say this....... feature flags should not be used in production level software......

Reply:
@sphinxc0re why not?